src/celastro/date.cpp

```diff
@@ -470,7 +463,7 @@ TDBtoLocal(double tdb)
     d.wday = localt.tm_wday;
 #if defined(__GNUC__) && !defined(_WIN32)
     d.utc_offset = static_cast<int>(localt.tm_gmtoff);
-    d.tzname = localt.tm_zone;
+    strncpy(d.tzname, localt.tm_zone, sizeof(d.tzname)-1);
```

Comment:
can't remember if the initializer will zero out the array when it is created, maybe need to also set the last element to 0 just in case.